CONTRIBUTING.md

```diff
@@ -182,5 +182,13 @@ When squashing-and-merging:
 -   Do a final review of the one-line PR summary, ensuring that it meets the
     guidelines (e.g., "blob: add more blobbing") and accurately describes the
     change.
+-   Mark breaking changes with `BREAKING_CHANGE` in the commit message (e.g.,
+    "blob: BREAKING_CHANGE remove old blob").
+    -   If the PR includes a breaking change, it will be declared via a commit
+        with `BREAKING_CHANGE_OK` in it; Travis checks for this.
```

Comment:
I'm confused about "Travis checks for this", in light of the second bullet point which says it's not actually required. Does Travis checking for breaking changes and then not finding `BREAKING_CHANGE_OK` have consequences for the build?

Reply:
Travis verifies that if there's a breaking change, one of the commits in the PR has `BREAKING_CHANGE_OK` in it.

This text is talking about what we do during squash/merge, when all those commits get rolled up into one and there's free text for the commit message. There's no way to have Travis enforce this, but I want us as humans to including `BREAKING_CHANGE` in the squashed commit title if appropriate (and some judgement is required about that, thus the 2nd bullet).

HTH

Reply:
Yeah, that helps. It does feel a little weird to me for the first time this is mentioned is in the past tense, like, by the time one is following the steps in the squash-and-merge section the `BREAKING_CHANGE_OK` commit is already suppose to have occurred but there is no directive in the sections about making a PR or reviewing?

Reply:
Good point; added a bullet to the `Reviewer` section above.